extension/T5Integration/Camera.h

```diff
@@ -0,0 +1,67 @@
+#ifndef _WANDSERVICE_H
```

Comment:
Is this the intended name? Possible collision?

Reply:
Okay, I pushed a change to fix that